src/server/api/auxiaProxyRouter.ts

```diff
@@ -200,6 +200,29 @@ const guDefaultGateGetTreatmentsResponseData = (
     return data;
 };
 
+const isValidContentType = (contentType: string): boolean => {
+    const validTypes = ['Article'];
+    return validTypes.some((type: string): boolean => contentType === type);
```

Comment:
you can also use `validTypes.includes(contentType)`

Reply:
Done!